manager/src/main/java/org/apache/hertzbeat/manager/controller/AccountController.java

```diff
@@ -66,11 +68,14 @@ public class AccountController {
     @PostMapping("/form")
     @Operation(summary = "Account password login to obtain associated user information", description = "Account password login to obtain associated user information")
     public ResponseEntity<Message<Map<String, String>>> authGetToken(@Valid @RequestBody LoginDto loginDto) {
-        SurenessAccount account = accountProvider.loadAccount(loginDto.getIdentifier());
+        if (StringUtils.isBlank(loginDto.getIdentifier()) || StringUtils.isBlank(loginDto.getCredential())) {
+            return ResponseEntity.ok(Message.fail(MONITOR_LOGIN_FAILED_CODE, " identifier or credential is null"));
+        }
+        SurenessAccount account = accountProvider.loadAccount(loginDto.getIdentifier().trim());
         if (account == null || account.getPassword() == null) {
             return ResponseEntity.ok(Message.fail(MONITOR_LOGIN_FAILED_CODE, "Incorrect Account or Password"));
         } else {
-            String password = loginDto.getCredential();
+            String password = loginDto.getCredential().trim();
```

Comment:
hi, here we not suggest use the trim(), we donot need to update the username password the user input.  
StringUtils.isBlank is no need, because  the @Valid LoginDto has the vaild rule.